bcml/mergers/rstable.py

```diff
@@ -57,6 +57,8 @@ def calculate_size(
         size = getattr(calculate_size, "calculator").calculate_file_size_with_ext(
             data, wiiu=be, ext=ext, force=False
         )
+        if ext == ".bfres":

```

Comment:
This is being applied to the `rstb` calculation, which does not handle bfres. For bfres, size at this point will always be 0. This line then makes size 1000, causing it to fail line 76 and never trigger line 80, meaning every bfres gets exactly 1000 for its rstb entry.